src/abi.h

```diff
@@ -119,9 +121,17 @@ typedef enum {
     NETPLAN_AUTH_EAP_METHOD_MAX,
 } NetplanAuthEAPMethod;
 
+typedef enum {
+    NETPLAN_AUTH_PMF_MODE_DISABLED,
```

Comment:
thought: Do we want to have a `NETPLAN_AUTH_PMF_MODE_NONE` as a fallback/unset value?